website/www/site/content/en/documentation/programming-guide.md

```diff
@@ -499,16 +531,34 @@ the transform itself as an argument, and the operation returns the output
 [Output PCollection] = [Input PCollection] | [Transform]
 {{< /highlight >}}
 
+{{< highlight go >}}
+[Output PCollection] := beam.ParDo(s, [Transform], [Input PCollection])
```

Comment:
Suggested change:

```
[Output PCollection] := beam.ParDo(s, [Transform], [Input PCollection])
```

to

```
[Output PCollection] := beam.ParDo(scope, [Transform], [Input PCollection])
```

Reply:
Since this and the others are abstract, it's good to call out the scope with a less abstract name. Done.